htdocs/admin/company.php

```diff
@@ -534,7 +534,7 @@
 
 // Note
 print '<tr class="oddeven"><td class="tdtop"><label for="note">'.$langs->trans("Note").'</label></td><td>';
-print '<textarea class="flat quatrevingtpercent" name="note" id="note" rows="'.ROWS_5.'">'.(GETPOSTISSET('note') ? GETPOST('note', 'restricthtml') : $conf->global->MAIN_INFO_SOCIETE_NOTE).'</textarea></td></tr>';
+print '<textarea class="flat quatrevingtpercent" name="note" id="note" rows="'.ROWS_5.'">'.(!empty($conf->global->MAIN_INFO_SOCIETE_NOTE) ? $conf->global->MAIN_INFO_SOCIETE_NOTE : GETPOST("MAIN_INFO_SOCIETE_NOTE", 'restricthtml')).'</textarea></td></tr>';
```

Comment:
GETPOSTISSET seems to be lost here.
Note: For phpv8, to find more easyly regression, it is recommanded to send PR with very small PR, for example, 1 PR per file modified.

Reply:
@eldy ok it's ready for GETPOSTISSET
yes I understand, by some files have warnings that come from multiple files. But I will do otherwise in the future to limit the size of the PRs. Can you still accept this one because it would take too long to separate everything? I thank you